### PR TITLE
Rebrand Phase One: Legacy Design Tokens

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
These changes update our soon-to-be-legacy design tokens to match the brand updates ARC is implementing in Phase One.

Note: Color tokens (e.g. 200) do not match those in Figma, since the new foundations anticipate a different schema.

I also have not updated any MUI variables to use new tokens yet; this is purely a palette swap while we determine what is acceptable for ARC/Odyssey 0.3.0.

### CODE REVIEW

**Comment by edburyenegren-okta** [comment]
@nickarcidy-okta thanks! I'll get updates in tomorrow. 

Broad note that these mappings are being applied to our old tokens, which introduces some review difficulty. The biggest is that the `###` naming application doesn't line up. For example, in the current token schema, `900` is expected where we're using `700` in the new palette. This is why the values are mismatched today.

I will look at the ARC variables this week and see if we can smooth some of that over. It'd be easier for everyone if our naming was on the same page.

**Comment by nickarcidy-okta** [comment]
Thanks Edbury, as long as the values match the mapping as outlined, that's my top prio. Naming conventions I'm totally flexible on as far as alignment goes. If you, Jose, and/or Taylor have opinions there, I defer to you!

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 1]
```diff
@@ -8,105 +8,125 @@
...
         "100": {
           "comment": "Used for read-only/disabled states",
-          "value": "#ebebed"
+          "value": "#EDEDED"
+        },
+        "200": {
+          "comment": "Used for Display borders",
+          "value": "#E1E1E1"
         },
-        "200": { "comment": "Used for Display borders", "value": "#d7d7dc" },
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:17]: Figma has a Grey 300 as well, does that not exist in code currently?

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 2]
```diff
@@ -8,105 +8,125 @@
...
         "500": {
           "comment": "Used for Interactive UI borders",
-          "value": "#8c8c96"
+          "value": "#8D8D8D"
         },
         "600": {
           "comment": "Used for caption text, Status label, disabled Overlay Buttons",
-          "value": "#6e6e78"
+          "value": "#707070"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:27]: Should be: #6E6E6E

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 3]
```diff
@@ -8,105 +8,125 @@
...
+          "value": "#F2F3FD"
         },
         "300": {
           "comment": "Used for buttons disabled state fills & borders, focus rings",
-          "value": "#a7b5ec"
-        },
-        "400": {
-          "comment": "Used for a11y-friendly UI backgrounds",
-          "value": "#788ee2"
+          "value": "#C1C9F6"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:41]: 300 should be #9DAAF1

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 4]
```diff
@@ -8,105 +8,125 @@
...
         },
         "500": {
           "comment": "Used for links, buttons, icons, hover states, selected states",
-          "value": "#1662dd"
+          "value": "#546BE7"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:45]: 500 should be #4C64E1

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 5]
```diff
@@ -8,105 +8,125 @@
...
         },
         "900": {
           "comment": "Used for okta logo, brand moments, button hovers",
-          "value": "#00297a"
+          "value": "#2E40A5"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:49]: Blue 900 should be #22307C

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 6]
```diff
@@ -8,105 +8,125 @@
...
+        },
+        "300": {
+          "comment": "N/A",
+          "value": "#7BE09E"
         },
-        "300": { "comment": "N/A", "value": "#84d2b1" },
         "500": {
           "comment": "Used for moments of success, validation, wayfinding, status",
-          "value": "#00b478"
+          "value": "#16884A"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:63]: Green 500 should be #197F48

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 7]
```diff
@@ -8,105 +8,125 @@
...
         },
         "600": {
-          "comment": "Used for background color for success messaging",
-          "value": "#008055"
+          "comment": "N/A",
+          "value": "#158046"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:67]: Green 600 should be #0E562F

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 8]
```diff
@@ -8,105 +8,125 @@
...
         },
         "900": {
           "comment": "Used for moments of success, validation, wayfinding",
-          "value": "#00503c"
+          "value": "#0E562F"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:71]: Green 900 should be #0A4023

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 9]
```diff
@@ -8,105 +8,125 @@
...
         },
         "300": {
           "comment": "Used for danger button disabled state fills, borders, & focus ring",
-          "value": "#f88c90"
+          "value": "#FEBBAE"
         },
         "500": {
           "comment": "Used for system errors, danger buttons, error states, status dot",
-          "value": "#da372c"
+          "value": "#E72500"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:85]: Red 500 should be #D92300

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 10]
```diff
@@ -8,105 +8,125 @@
...
         },
         "900": {
           "comment": "Used for destructive button & invalid input hovers",
-          "value": "#640019"
+          "value": "#951800"
```
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:89]: Red 900 should be #711200

**Review by taylorridgway-okta — CHANGES_REQUESTED** [review]

File: packages/odyssey-design-tokens/src/color/palette.json (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/odyssey-design-tokens/src/color/palette.json:17]: Correct, there's no `300` in the current tokens.

File: packages/odyssey-design-tokens/src/color/palette.json [hunk 11]
```diff
@@ -8,105 +8,125 @@
...
       "red": {
         "000": {
           "comment": "Used for system errors, danger buttons, error states",
-          "value": "#ffdfe2"
+          "value": "#FFF0EE"
         },
         "300": {
           "comment": "Used for danger button disabled state fills, borders, & focus ring",
-          "value": "#f88c90"
+          "value": "#FEBBAE"
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-design-tokens/src/color/palette.json:81]: Are the capital letters easier to copy outta Figma? :P

File: packages/odyssey-design-tokens/src/font.json [hunk 12]
```diff
@@ -3,11 +3,11 @@
     "family": {
       "base": {
         "comment": "Default font used within all Okta product user interfaces",
-        "value": "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif"
+        "value": "'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif"
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-design-tokens/src/font.json:6]: Do we provide `Inter` in our library or is it something apps are expected to have?

File: packages/odyssey-storybook/src/guidelines/Colors.stories.mdx [hunk 13]
```diff
@@ -4,6 +4,8 @@ import { Meta } from "@storybook/addon-docs/blocks";
 
 # Color
 
+THIS IS A FAKE CHANGE IN ORDER TO TRIGGER A DEPLOYMENT.
+
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/guidelines/Colors.stories.mdx:8]: You can remove this now right? Or you mean deploying an update to this file?

**Review by KevinGhadyani-Okta — APPROVED** [review]
Just some questions. *NON-BLOCKIING*.

File: packages/odyssey-design-tokens/src/color/palette.json (hunk 11 above)
Comment (edburyenegren-okta) [reply packages/odyssey-design-tokens/src/color/palette.json:81]: I'm going to downcase all of these before merge. 

Right now, it's useful for me to see which values are deviations and match them against Figma's output.

File: packages/odyssey-design-tokens/src/font.json (hunk 12 above)
Comment (edburyenegren-okta) [reply packages/odyssey-design-tokens/src/font.json:6]: We don't provide `Inter`, so this stack will immediately fall back to system fonts if the downstream teams haven't provided it / it's not available locally.

File: packages/odyssey-storybook/src/guidelines/Colors.stories.mdx (hunk 13 above)
Comment (edburyenegren-okta) [reply packages/odyssey-storybook/src/guidelines/Colors.stories.mdx:8]: I'll be removing this before merge. We need to update https://github.com/okta/odyssey/blob/develop/.github/workflows/ci.yml#L40 to include `odyssey-design-tokens` (possibly others).

**Review by ganeshsomasundaram-okta — APPROVED** [review]

**Review by taylorridgway-okta — APPROVED** [review]

**Pull request merged**